frontend/app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -102,16 +102,24 @@ const loading = computed<boolean>(() => {
 const balanceBlockchainRoute = Routes.ACCOUNTS_BALANCES_BLOCKCHAIN.route;
 </script>
 <style scoped lang="scss">
-.blockchain-balance-box {
-  &__icon {
-    filter: grayscale(100%);
-    margin: 0;
-    margin-right: 5px !important;
-    width: auto !important;
-  }
+:deep() {
+  .blockchain-balance-box {
+    &__icon {
+      filter: grayscale(100%);
+      margin: 0;
+      margin-right: 5px !important;
+      width: auto !important;
+    }
 
-  &__item:hover &__icon {
-    filter: grayscale(0);
+    &__item {
+      &:hover {
+        .blockchain-balance-box {
+          &__icon {
+            filter: grayscale(0);
+          }
+        }
+      }
+    }
```

Comment:
Is there any reason we need `deep` here? 
From what I remember we use deep to affect global classes

Reply:
I want it to also affect the blockchain balances details (Loopring etc)

Reply:
OK but ideally since we control both components we could actually add rules to that component too instead of using `deep` couldn't we?